src/lib/version/version.h

```diff
@@ -99,12 +99,41 @@ static inline const char *px4_build_uri(void)
 }
 
 /**
+ * Convert a version tag string to a number
+ * @param tag version tag in one of the following forms:
+ *            - dev: v1.4.0rc3-7-g7e282f57
+ *            - rc: v1.4.0rc4
+ *            - release: v1.4.0
+ *            - linux: 7.9.3
+ * @return version in the form 0xAABBCCTT (AA: Major, BB: Minor, CC: Patch, TT Type @see FIRMWARE_TYPE)
+ */
+__EXPORT uint32_t version_tag_to_number(const char *tag);
+
+/**
  * get the PX4 Firmware version
  * @return version in the form 0xAABBCCTT (AA: Major, BB: Minor, CC: Patch, TT Type @see FIRMWARE_TYPE)
  */
 __EXPORT uint32_t px4_firmware_version(void);
 
 /**
+ * Convert a version tag string to a vendor version number
+ * @param tag version tag in one of the following forms:
+ *            - vendor: v1.4.0-0.2.0
+ *            - dev: v1.4.0rc3-7-g7e282f57
+ *            - rc: v1.4.0rc4
+ *            - release: v1.4.0
+ *            - linux: 7.9.3
+ * @return version in the form 0xAABBCC (AA: Major, BB: Minor, CC: Patch)
```

Comment:
Can you please change the return type to  `0xAABBCCTT` to make it consistent with `version_tag_to_number` (even if TT is just constant)?

Reply:
Done in 52f6c1e